src/util/string.cpp

```diff
@@ -729,11 +729,11 @@ static bool parseNamedColorString(const std::string &value, video::SColor &color
 	return true;
 }
 
-std::wstring removeChatEscapes(const std::wstring &s) {
+std::wstring removeEscapes(const std::wstring &s) {
 	std::wstring output;
 	size_t i = 0;
 	while (i < s.length()) {
```

Comment:
I feel like this function could be structured better.  Also, this deserves a unit test.